Sources/MongoSwift/MongoDatabase.swift

```diff
@@ -250,7 +251,8 @@ public struct MongoDatabase {
      *   - options: Optional `ListCollectionsOptions` to use when executing this command
      *   - session: Optional `ClientSession` to use when executing this command
      *
-     * - Returns: An array of `MongoCollection`s that match the provided filter.
+     * - Returns: An `EventLoopFuture<[MongoCollection<Document>]>`s containing collections that match the provided
```

Comment:
can remove the "s" after `EventLoopFuture<...>`

Reply:
fixed.